src/style.css

```
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background: url(images/star.svg) no-repeat;
```

Fix this line, run the test again, and deploy changes. I gave u this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/background-position) that u add `background-position: center;`

```suggestion
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
```

---

I understand now, I thought you ask me write shortly %)
Fixed it.